CMakeLists.txt

```diff
@@ -33,16 +33,16 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 option(SKIP_PACKAGING "" OFF)
 
 # enable frontend build
-option(USE_FRONTEND "build the browser-based frontend (requires yarn and nodejs)" ON)
+option(USE_FRONTEND "build the browser-based frontend(requires yarn and nodejs)" ON)
```

Comment:
(nitpick) It seems strange that the formatter deleted a whitespace in a string (there are a lot of similar occurences in this file, also in comments)

Reply:
Lol, you are right. I will fix